src/timeline-manager.tsx

```diff
@@ -65,6 +66,8 @@ class TimelineManager {
 
   public loadMedia = () => {
     this._listenerDuration();
+    // @ts-ignore
+    this._player.ready().then(() => this._isPreventSeekActive = this._player.preventSeekOptions.isActive)
```

Comment:
what if this._player.preventSeekOptions is updated at runtime ? maybe it should fire an event instead ?

Reply:
this._player.preventSeekOptions is updated on player side after the new plugin is firing an event (which happens only inside loadmedia).

Reply:
so why not listen to the event here ?